src/functions.js

```diff
@@ -121,7 +120,27 @@ var _ = Mavo.Functions = {
 			return n && n > 1? [arr] : arr;
 		}
 
-		return n && n > 1? arr.slice(0, n) : arr[0];
+		if (n && n > 1) {
```

Comment:
Sorry, I didn't see this before. `n > 1` suffices here, no need for `n` as well. If `n` is greater than 1, it's definitely a positive number. (null etc all would return false).

One case to think about: Should we return an empty array or `null` if n is 0?